homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -116,10 +116,12 @@ def get_state(self, daytime_date, after_shkia_date, after_tzais_date):
             # Compute the weekly portion based on the upcoming shabbat.
             return after_tzais_date.upcoming_shabbat.parasha
         if self._type == "holiday":
-            self._holiday_attrs["id"] = after_shkia_date.holiday_name
-            self._holiday_attrs["type"] = after_shkia_date.holiday_type.name
-            self._holiday_attrs["type_id"] = after_shkia_date.holiday_type.value
-            return after_shkia_date.holiday_description
+            self._holiday_attrs = {
+                "id": after_shkia_date.holiday_name,
+                "type": after_shkia_date.holiday_type.name,
+                "type_id": after_shkia_date.holiday_type.value,
+            }
+            return after_shkia_date.holiday_description or None
```

Comment:
this is a breaking change, since possible return value is changed, those existing automations may be broken

Reply:
You are right, maybe for a future PR